### DISCUSSION TITLE
RTMP streaming app for Android

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3730] [General Owncast discussion]
Hello again.

Is there a recommended Android app to transmit from your cell phone? I found one in the Play Store but it only allows 5 minutes of airtime, unless it lasts an hour.

Thank you

**Comment by ghost** [comment]
I've used Larix Broadcaster in the past for this. https://softvelum.com/larix/

**Reply by gabek** [reply]
I've used Larix as well, and it works.

**Comment by delta0815** [comment]
After searching a lot I finally found a free alternative to Larix:

https://play.google.com/store/apps/details?id=app.irlpro.android&hl=de&pli=1

I have no idea why this isn't featured anywhere as it seems to have way more features and a better interface than any free alternative out there...

**Reply by gabek** [reply]
If you're interested in adding it to our broadcasting software documentation, a PR would be appreciated! https://owncast.online/docs/broadcasting/

**Reply by delta0815** [reply]
Sure thing. Will do tonight! :-)

**Comment by gui-lux** [comment]
Hi,
I've tested this android app called "IP Webcam" against my owncast instance and it just worked. Tested on personal Wifi so far but I guess it will work fine with acceptable mobile network coverage. (shall give a try)
I used to run this never updated app a lot with obs, but installing apk updater recently made me slip into its latest version, and appreciate its recent RTMP support.
Unfortunately, this app is not on f-droid. https://play.google.com/store/apps/details?id=com.pas.webcam
Can't wait to watch mobile streamers :)

**Reply by fsan2val** [reply]
Thanks for putting this list together.

I tried to check several of the apps mentioned, but some of them cannot be tested because the download links are broken or the apps are no longer available.

Just wanted to point this out in case the list can be updated, which would help other users looking for currently available RTMP apps.

Thanks for the contribution.

**Reply by gui-lux** [reply]
Thank you for the heads up.
The link to the app is now working (can't figure out what went wrong)

**Comment by freshpixelmanuel-cyber** [comment]
Hey! I just released RtmpCaster, a free and open Android app for RTMP screen streaming — might be exactly what you're looking for as a Larix replacement.
What it does:

Streams your Android screen directly to Twitch, YouTube, or any RTMP server
Configurable resolution (480p–1080p), bitrate and FPS
No ads, no registration, no hidden costs
Privacy-first: all settings stored locally on device

How it works:

Enter your RTMP URL + stream key in settings
Tap "Start Stream"
Confirm screen capture permission
You're live!

Note: On Android 14+ the system shows a screen capture dialog — that's an Android security requirement, not the app.
Still in beta, feedback very welcome!
🔗 Download (free): https://play.google.com/apps/internaltest/4701521557020617455
🌐 Info: https://www.manuelfischer.com/rtmpcaster/
💬 Discord: https://discord.gg/egHXwDcbcA

**Reply by fsan2val** [reply]
Thank you for sharing this information.

I reviewed several of the applications listed here, but I was unable to test some of them because the links are broken or the applications no longer seem to exist.

I am sharing this feedback as a contribution to the discussion. It would be very helpful if the list could be reviewed and updated when possible, as many users rely on it to find working RTMP applications.

Thank you for your efforts and for maintaining this resource.

**Comment by muzammilijaz671-a11y** [comment]
Android users interested in mobile streaming often need reliable apps that support RTMP and work smoothly with different broadcasting platforms. This GitHub discussion explores several Android streaming options and their practical use. For readers also exploring [Xuper TV](https://xupertvz.co/), it provides useful context around mobile streaming, Android apps, and digital entertainment.

**Reply by gabek** [reply]
omfg shut up stupid AI bot @muzammilijaz671-a11y.
